framework/pym/play/commands/junit.py

```diff
@@ -70,25 +68,19 @@ def junit(app, args):
     # Run app
     test_result = os.path.join(app.path, 'test-result')
     if os.path.exists(test_result):
+        print("~ cleaning test results")
         shutil.rmtree(test_result)
     args.append('-Dplay.autotest')
     java_cmd = app.java_cmd(args, className='play.test.Runner')
     try:
-        process = subprocess.Popen (java_cmd, env=os.environ)
+        process = subprocess.Popen(java_cmd, env=os.environ)
         signal.signal(signal.SIGTERM, handle_sigterm)
         signal.signal(signal.SIGINT, handle_sigint)
         return_code = process.wait()
         if 0 != return_code:
+            print("~ java process exited with exit code %d" % return_code)
```

Comment:
Important bit, logging return code to be picked up by CI

Reply:
very minor: "java process exited with code %"?